.github/workflows/ci-push.yml

```diff
@@ -190,3 +190,36 @@ jobs:
         name: racketcs-${{ matrix.os }}-x64_git${{ github.sha }}
         path: ../racketcs-${{ matrix.os }}-x64_git${{ github.sha }}.tar.bz2
 
+# Tests
+  test-cgc:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-18.04, macos-latest]
+        cify: [cify, nocify]
+
+    runs-on: ${{ matrix.os }}
+
+    needs: build-racketcgc
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/download-artifact@master
+        with:
+          name: racketcgc-${{ matrix.os }}-${{ matrix.cify }}-x64_git${{ github.sha }}
+          path: ../
+      - name: Untar
+        working-directory: ../
+        run: tar -xvjf racketcgc-${{ matrix.os }}-${{ matrix.cify }}-x64_git${{ github.sha }}.tar.bz2
+      - name: Extend PATH with Racket executable
+        working-directory: ../
+        run: echo "::set-env name=PATH::$PWD/racketcgc/bin:$PATH"
+      - name: Check for Racket
+        run: racket --version
+      - name: Install catalog and required packages
+        run: |
+          racket -l- pkg/dirs-catalog --immediate $PWD/rktcat $PWD/pkgs/
```

Comment:
Can't you do this with `make PKGS="racket-test"`?

Reply:
Good question - isn't that incantation to build the whole racket? Once we have racket built, we can just add those dirs to the catalog, no?